.github/workflows/test-ci-reusable.yml

```diff
@@ -10,11 +10,18 @@ on:
       ref:
         required: false
         type: string
+      sha:
+        required: true
+        type: string
       
 env:
   OPENSUSE_UNOFFICIAL_LIBCONTAINERS_KEY_URL: "https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/unstable/xUbuntu_22.04/Release.key"
   OPENSUSE_UNOFFICIAL_LIBCONTAINERS_SOURCE_URL: "https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/unstable/xUbuntu_22.04"
 
+permissions:
+  statuses: write
+  packages: write
```

Comment:
Does this need the `packages` permission to work?

Reply:
Hmm, alright. So that was just missing from this workflow from before, not needed by these new step additions?

Reply:
I believe so? But I am noticing that previous tests weren't failing ... I'll double check

Reply:
oh they could be passing without the manually added permissions because workflow permissions for cryostatio/cryostat-operator are set to read and write

Reply:
I know we opted for manual write permissions in Cryostat, but I'm unsure if that carried over to Cryostat-operator

Reply:
I think that's something we should keep consistent across all the repos as a general policy.

Reply:
ok then yes, the package permissions should stay :+1:

Reply:
@mwangggg @aali309 @ebaron I'm going to flip this switch at the organization level now so that workflows need to specifically request write permissions. I'm sure this will expose some other CI configurations that don't yet make that request, so things are likely to start failing but just need a simple patch-up like this one.